### PR TITLE
Related And Comparison

### PR DESCRIPTION
**Pull request by theGuyNextDoor**
Very minimal CSS so far

Changes other than the Related and Comparison components :
1) styles.css, path client>public>styles.css
2) <link /> to styles.css in html file
3) Uncommented <RelatedAndComparison /> in App.js

Related And Comparison additions:
1) Renders correct related products.
2) renders correct products from your outfits.
3) adds products to your outfits from related outfits and re-renders.
4) deletes products from related outfits and re-renders.

### CODE REVIEW

**Comment by theGuyNextDoor** [comment]
Looking for a code review, please and thank you. For more info see initial comment above.

File: client/src/components/RelatedAndComparison/RelatedAndComparison.jsx [hunk 1]
```diff
@@ -1,16 +1,123 @@
...
+  const [relatedStyles, setRelatedStyles] = useState([]);
+  const [relatedProducts, setRelatedProducts] = useState([]);
+  const [styles, setStyles] = useState([]);
+  const [yourOutfit, setYourOutfit] = useState([]);
+
+  useEffect(() => {
+    axios.get('/api/products/48432/related')
+      .then(({ data }) => {
+        setRelatedIds(data);
+      });
```
Comment (jacquesdubois) [inline client/src/components/RelatedAndComparison/RelatedAndComparison.jsx:19]: When you're ready to refactor to useContext, consider replacing the hard-coded '48432'  in the url with state.selectedProduct

Comment (jacquesdubois) [reply client/src/components/RelatedAndComparison/RelatedAndComparison.jsx:19]: You won't need to replace any of your useState hooks, btw. All of those can manage local state while you read global state from StateContext

File: client/src/components/RelatedAndComparison/RelatedAndComparison.jsx [hunk 2]
```diff
@@ -1,16 +1,123 @@
...
+            images: resultsIdx.photos,
+          };
+          relatedStylesFormat.results.push(thisIn);
+        }
+        return relatedStylesFormat;
+      }));
+    Promise.all(relatedStylesData)
+      .then((values) => {
+        setRelatedStyles(values);
+      });
```
Comment (jacquesdubois) [inline client/src/components/RelatedAndComparison/RelatedAndComparison.jsx:42]: Good use of Promise.all here

File: client/src/components/RelatedAndComparison/RelatedAndComparison.jsx [hunk 3]
```diff
@@ -1,16 +1,123 @@
...
+  }, [relatedIds]);
```
Comment (jacquesdubois) [inline client/src/components/RelatedAndComparison/RelatedAndComparison.jsx:43]: Good use of Promise.all here. During optimization, we may be able to raise these axios requests to the app level

**Review by jacquesdubois — APPROVED** [review]
Looks great overall

**Pull request merged**